.github/container/Dockerfile.axlearn

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1-labs
 ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
-ARG URLREF_AXLEARN=https://github.com/Steboss/axlearn.git#main
+ARG URLREF_AXLEARN=https://github.com/Steboss/axlearn.git#sbosisio/tree_util
```

Comment:
I was confused why this was enough...

I think https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/container/manifest.yaml needs an `axlearn` entry, so the `bump-manifest` job (e.g. https://github.com/NVIDIA/JAX-Toolbox/actions/runs/14511741187/job/40711854578?pr=1396) will fetch the latest commit hash. Then you can plumb it through like this: https://github.com/NVIDIA/JAX-Toolbox/blob/88891cf7980e5d658aff95b8ab2cae3fa1200d82/.github/workflows/_ci.yaml#L62-L66